design-toolkit/components/src/components/text-area-field/text-area-field.test.tsx

```diff
@@ -52,7 +52,7 @@ describe('TextAreaField', () => {
     setup({
       size: 'small',
       description,
-      label: 'Label',
+      'aria-label': 'Label',
```

Comment:
Rather than doing this, I'd like to take the approach that we did with DateField, where the label prop feeds into the aria-label prop: https://github.com/gohypergiant/standard-toolkit/blob/main/design-toolkit/components/src/components/date-field/index.tsx#L120

Reply:
I think I've understood.